app/src/ui/remove-repository/confirm-remove-repository.tsx

```diff
@@ -45,9 +66,26 @@ export class ConfirmRemoveRepository extends React.Component<
             }"?
           </p>
           <p className="description">
-            The repository will be removed from GitHub Desktop but will remain
-            on disk.
+            The repository will be removed from GitHub Desktop.
           </p>
+
+          <div>
+            <p>
+              <strong>
```

Comment:
Not sure how I feel about this part. Here's how it looks in the rest of the dialog:

Reply:
I can have it behave either way the team prefers.

Reply:
Let's see how having _just the checkbox and message_ feels. Add an updated screenshot to this PR and we'll see how the team feels...

Reply:
Current: (as of this PR) 
![image](https://user-images.githubusercontent.com/7467062/41672719-41513b26-7480-11e8-8c52-32b46c821b29.png)

Proposed:
![image](https://user-images.githubusercontent.com/7467062/41672704-370724fa-7480-11e8-871f-d502ceadbf58.png)

Reply:
Maybe another option?
![image](https://user-images.githubusercontent.com/7467062/41672850-a11b902e-7480-11e8-868a-5f999bb9dc4b.png)

Reply:
I'm in favor of this direction: ![remove](https://user-images.githubusercontent.com/7467062/41672704-370724fa-7480-11e8-871f-d502ceadbf58.png)

I don't think it's necessary to try to emphasize the option to move given that it's not a destructive action in case the person misses that checkbox.